db/migrate/20230606182212_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) given, the column which references the user on the comments table should be `author_id` not `user_id`. Please implement this change 👍

Reply:
![Capture](https://github.com/raihan2bd/blog-app/assets/35267447/9d771f96-47a9-494a-9dcb-05a3711fd087)

Thanks a lot for your suggestion! I already implemented it according to the diagram, but I'm happy to make changes to make it match your vision. 👍